languages.toml

```
@@ -1493,7 +1493,16 @@ name = "dockerfile"
 scope = "source.dockerfile"
 injection-regex = "docker|dockerfile"
 roots = ["Dockerfile", "Containerfile"]
-file-types = [{ glob = "Dockerfile*" }, { glob = "dockerfile*" }, { glob = "Containerfile*" }, { glob = "containerfile*" }]
+file-types = [
+  "Dockerfile",
+  { glob = "Dockerfile*" },
```

Let's split this up further actually:

```suggestion
  "Dockerfile",
  { glob = "Dockerfile" },
  { glob = "Dockerfile.*" },
```

The second entry will only match files fully named `Dockerfile`. So we would match `DockerfileFooBar` for example

---

Hmm, from what I remember when I was testing the patterns, `Dockerfile*` did match `Dockerfile.temp`. I think between the extension check, `Dockerfile`, and `Dockerfile.*`, it would cover any kind of dot dockerfile pattern. Though I may be mistaken in that.

---

Right yeah, I mean the second pattern in the suggestion (`{ glob = "Dockerfile" }`)